File: chapter2/namedArray.js
/*
 * Read a JSON file with multiple named array elements,
 * and display a value from each element.
 */
var fs = require ("fs");

// read the file synchronously, convert to a JSON object
var content = fs.readFileSync ("namedArray.json");
var data = JSON.parse (content);

// display the first server address
console.log ("server");
console.log ("  name: " + data.servers[0].name);
console.log ("  address: " + data.servers[0].address);
// display the second server address
console.log ("\nserver");
console.log ("  name: " + data.servers[1].name);
console.log ("  address: " + data.servers[1].address);

File: chapter2/namedArray.py
"""
Read a JSON file with multiple named array elements,
and display a value from each element.
"""
from json import loads

# read the file and convert to a JSON object
content = open ("namedArray.json", "rU").read ()
data = loads (content)

# display the first server address
print ("server")
print ("  name: " + data["servers"][0]["name"]) 
print ("  address: " + data["servers"][0]["address"])
# display the second server address
print ("\nserver")
print ("  name: " + data["servers"][0]["name"]) 
print ("  address: " + data["servers"][0]["address"])
